### PR TITLE
Add triaging document

### PR DESCRIPTION
**Pull request by mirceaulinic**
<!-- Make sure you have read http://napalm.readthedocs.io/en/latest/contributing/index.html --!>

### CODE REVIEW

File: docs/development/triage.rst [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+.. _triage-duplicate-label:
+
+``duplicate``
+-------------
+
+Applicable to Issues only, to be added before closing a duplicate.
+
+.. _triage-epic-label:
+
+``epic``
```
Comment (dbarrosop) [inline docs/development/triage.rst]: We should probably eliminate this. We can use the github projects instead if needed but zenhub is probably dead now we don't have multiple repos.

File: docs/development/triage.rst [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+.. _triage-version-milestone:
+
+Version
+-------
+
+The Issue will be solved, or the PR will be included in this release.
+
+.. _triage-approved-milestone:
+
+``APPROVED``
```
Comment (dbarrosop) [inline docs/development/triage.rst]: We should move this to github projects instead.

File: docs/development/triage.rst [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+
+``DISCUSSION``
+--------------
+
+The Issue or the PR needs further discussion.
+
+
+.. _triage-pipeline:
+
+Pipeline
```
Comment (dbarrosop) [inline docs/development/triage.rst]: github projects instead?

**Comment by dbarrosop** [comment]
I'd also add the following labels:

1. core/base - When something affects things under `base`, for example the validate functionality, the testing framework, get_network_driver, etc...
2. documentation - When a change in the docs are requested

File: docs/development/triage.rst (hunk 2 above)
Comment (mirceaulinic) [reply docs/development/triage.rst]: Milestones are a GitHub feature, not ZenHub, and it helps to schedule a issue/PR to a certain release version.

**Comment by dbarrosop** [comment]
Something that occurred to me, do we want a "method_implementation" label?

**Comment by dbarrosop** [comment]
I think we should merge this one. We can revisit it later on if needed :) Feel free to update the branch and merge.

**Pull request merged**